builtin/providers/aws/resource_aws_elasticache_cluster.go

```diff
@@ -192,6 +192,12 @@ func resourceAwsElasticacheCluster() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"replication_group_id": &schema.Schema{
```

Comment:
In this PR, the ElasticacheReplicationGroup don't use the ElasticacheCluster resource. Is there any reason to do that ?!
I might miss something ;-)